c3/utils/utils.py

```diff
@@ -50,17 +50,23 @@ def replace_symlink(path: str, alias: str) -> None:
         os.remove(alias)
     except FileNotFoundError:
         pass
+    except PermissionError:
+        warnings.warn("Could not remove symlink")
     try:
         os.symlink(path, alias)
     except FileExistsError:
         pass
     except OSError:
-        Warning("OSError encountered while creating symlink")
+        warnings.warn("OSError encountered while creating symlink")
+    except PermissionError:
```

Comment:
The `OSError` above is expected to subsume this `PermissionError` which would never be reached in that case and is possibly redundant.